src/llm/apis/openai_completions.cpp

```diff
@@ -166,7 +161,25 @@ static absl::Status downloadImage(const char* url, std::string& image, const int
     return absl::OkStatus();
 }
 
-absl::Status OpenAIChatCompletionsHandler::parseMessages() {
+ov::Tensor loadImageStbiFromMemory(const std::string& imageBytes) {
+    int x = 0, y = 0, channelsInFile = 0;
```

Comment:
How is `channelsInFile` used? It's hardcoded here along with `x` and `y`, but other values are passed to `loadImageStbi`